Blades in the Dark/translations/es.json

```diff
@@ -28,7 +28,7 @@
     "an_upgrade": "Una mejora",
     "arcane_implements": "Utensilios Arcanos",
     "arcane_implements_description": "Un frasco de mercurio. Una bolsa de sal negra. Un ancla espiritual en forma de pequeña piedra. Una botella espiritual. Un frasco de electroplasma diseñado para romperse y rociarlo todo en el impacto.",
-    "armor": "Protección",
+    "armor": "Estándar",
```

Comment:
Is this intentional?

Reply:
If you mean the change of Protección/Estándar, yes It's intentional. That was a previous bad translation. There was some things I didn't know exactly what they were until I've seen It in the sheet.